src/prototype_creep_harvest.js

```diff
@@ -49,8 +49,7 @@ Creep.prototype.spawnCarry = function() {
   }
 
   var foundKey;
-  let sizes = Object.keys(config.carry.sizes).sort(function(a, b) { return parseInt(a, 10) - parseInt(b, 10); });
-  for (let key of sizes) {
+  for (let key in config.carry.sizes) {
```

Comment:
yep. `for (let key of Object.keys(config.carry.sizes))` is good aswell. `Object.keys` will enumerate only own properties, and so it do not trigger warning in codeclimate (in this it's suppressed, but logicaly warning should still be there)